frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -375,14 +383,25 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               onChange={React.useCallback((diskBus) => setBus(DiskBus.fromString(diskBus)), [
                 setBus,
               ])}
+              isValid={!isValidationError(busValidation)}
               value={asFormSelectValue(bus)}
               id={asId('interface')}
               isDisabled={inProgress}
             >
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
-              {DiskBus.getAll().map((b) => (
-                <FormSelectOption key={b.getValue()} value={b.getValue()} label={b.toString()} />
-              ))}
+              {!(allowedBusses || new Set(DiskBus.getAll())).has(bus) && (
+                <FormSelectOption
+                  isDisabled
```

Comment:
sorry it does not work with isDisabled - lets remove it

Reply:
Are you sure? for me it does:
![Screenshot from 2020-01-22 21-42-53](https://user-images.githubusercontent.com/18301938/72928229-44e16b80-3d60-11ea-9def-450ec063f383.png)

Reply:
for me it does not

table:

![aa](https://user-images.githubusercontent.com/3648838/72977506-fe7a2400-3dd4-11ea-98f4-18a524ce8fc2.png)

dialog after opening:

![aaa](https://user-images.githubusercontent.com/3648838/72977547-0a65e600-3dd5-11ea-9d02-16e1d501bf49.png)